specs/language/introduction.tex

```diff
@@ -77,46 +118,67 @@
 architecture and the way they relate to the \acrshort{spmd} program model. In
 this document we will use the terms as defined in the following subsections.
 
-\Sub{\gls{lane}}{Intro.Model.Lane}
+\Sub{\acrshort{spmd} Terminology}{Intro.Model.Terms}
 
-\p A \gls{lane} represents a single computed element in an \acrshort{spmd}
+\Par{Host and Device}{Intro.Model.Terms.HostDevice} \acrshort{hlsl} is a
+data-parallel programming language designed for programming auxiliary processors
+in a larger system. In this context the \textit{host} refers to the primary
+processing unit that runs the application which in turn uses a runtime to
+execute \acrshort{hlsl} programs on a supported \textit{device}. There is no
+strict requirement that the host and device be different physical hardware,
+although they commonly are. The separation of host and device in this
+specification is useful for defining the execution and memory model as well as
+specific semantics of language constructs.
+
+\Par{\gls{lane}}{Intro.Model.Terms.Lane}
+A \gls{lane} represents a single computed element in an \acrshort{spmd}
 program. In a traditional programming model it would be analogous to a thread of
 execution, however it differs in one key way. In multi-threaded programming
 threads advance independent of each other. In \acrshort{spmd} programs, a group
 of \gls{lane}s execute instructions in lock step because each instruction is a
 \acrshort{simd} instruction computing the results for multiple \gls{lane}s
```

Comment:
wording suggestion:

> In \acrshort{spmd} programs, a group of \gls{lane}s share a common instruction stream, which they execute in lockstep.

Note: I believe "lockstep" is commonly used as a single word now, although it is occasionally hyphenated also. This wording also includes instructions that don't "compute results" (e.g. barrier instructions).